README.md

```diff
@@ -1 +1,194 @@
-# book-an-appointment
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h1><b> 🏖 Motorbike Booking 🏞 </b></h1>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [💸 Motorbike Booking ](#-Motorbike-booking-)
+- [🛠 Built With ](#-built-with-)
+  - [📌 Tech Stack ](#-tech-stack-)
+- [🎲 Key Features ](#-key-features-)
+- [🚀 Live Demo](#-live-demo-)
+- [💻 Getting Started ](#-getting-started-)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors ](#-authors-)
+- [🔭 Future Features ](#-future-features-)
+- [🤝 Contributing ](#-contributing-)
+- [⭐️ Show your support ](#️-show-your-support-)
+- [🙏 Acknowledgments ](#-acknowledgments-)
+- [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# Motorbike Booking <a name="about-project"></a>
+
+**Motorbike Booking** is a web application that offers a diverse selection of luxurious accommodations and amenities. Find your slice of paradise, effortlessly plan your trip, and make memories that last a lifetime with MotorbikeHub.
+
+
+# 🛠 Built With <a name="built-with"></a>
+
+## 📌 Tech Stack <a name="tech-stack"></a>
+
+| Domain | Tech Stack |
+|--------|--------|
+| Server | <a href="https://github.com/facebook/create-react-app">React</a> |
+| Library | <a href="https://redux.js.org/">Redux</a> |
+| Compiler | <a href="https://babeljs.io/docs/">Babel</a> |
+
+<!-- Features -->
+
+# 🎲 Key Features <a name="key-features"></a>
+
+- Create a new account
+- Log in to an existing account
+- Sign out from a session
+- Choose from a variety of beatiful Motorbikes
+- Book a Motorbike 
+- Add a Motorbike
+- Delete a Motorbike 
+
+# 🚀 Live Demo <a name="live-demo"></a>
+
+👉 [Not Available](https://....)
+
+# 💻 Rails API <a name="rails-api"></a>
+
+```sh
+  https://github.com/MenendezON/book-an-appointment
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- GETTING STARTED -->
+
+# 💻 Getting Started with Create React App <a name="getting-started"></a>
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/MenendezON/book-an-appointment.git
+```
+
+To install dependencies, run:
+
+```sh
+npm install
+```
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### `npm test`
```

Comment:
- [ ] This 2 commands "npm start" and "npm test" won't work, since your app needs to be started with "rails server", and you have not included any frontend tests.

Also, those script have not been included in `package.json`

Kindly, remove those 2 commands from your readme since they won't work. 👍